recipes/intervene/meta.yaml

```diff
@@ -48,8 +48,6 @@ test:
     - intervene.modules.pairwise
     - intervene.modules.upset
     - intervene.modules.venn
-  commands:
-    intervene --help
```

Comment:
There was just a typo, i.e., missing dash, since `commands` is supposed to be a list. Use:
```yaml
  commands:
    - intervene --help
```
If the program has an entry point `intervene`, then including it in `test/commands`, like you did before, was indeed a good thing to do.

Reply:
@mbargull thanks for catching the typo.